### DISCUSSION TITLE
Recon live not working.

### DISCUSSION THREAD

**Discussion by Morthawt** [xchwarze/wifi-pineapple-cloner#27] [Q&A] [answered]
Hi. I have a gl.inet 750 "Creta" and it is working great, except the live recon view is not viewing. It is writing to disk, but since the only reason I use my pineapple is to watch what is in my area, it's a big issue having to stop and then load the file, all the while missing things because it's no longer scanning. This issue seems to have been known about for a while on the actual hardware: https://forums.hak5.org/topic/51020-live-results-not-working-with-262/

Is there any mitigation to resolve this and get the live recon view working?

**Comment by xchwarze** [comment, marked as answer]
text from downloads docs: https://gitlab.com/xchwarze/wifi-pineapple-cloner-builds#supported-devices
```
As tetra is made to be used on hardware with 32 MB of flash I had to cut some dependencies from the default installation.
These dependencies will be installed automatically when the pinapple is connected to the internet and booting.
If you want to manually run this process wpc-tools missing_packages
Without these dependencies you will not be able to use the live scan type and some modules. However you will be able to use the timed scans and the rest of the tools.
```
I understand that this is what is happening to you. having a usb installed and running this command would solve it.

**Reply by Morthawt** [reply]
Thanks. I had to do another factory reset and then that command worked. Working great now. Thanks for the help! 🍻
